src/visitors/components/signup_form.js

```diff
@@ -73,37 +73,37 @@ const signupForm = (props) => {
                 label="Full Name"
                 name={`fullname$${props.uuid}`}
                 type="text"
-                error={props.errors.formSender && VISITOR_NAME_INVALID}
+                error={props.errors.name && VISITOR_NAME_INVALID}
                 required
               />
               <LabelledInput
                 id="visitor-signup-email"
                 label="Email Address"
                 name={`email$${props.uuid}`}
                 type="email"
-                error={props.errors.formEmail}
+                error={props.errors.email}
```

Comment:
`formEmail` is used in the `409` error case in the `pages/Signup.js` component, so needs to be updated to work with this change